src/tribol/interface/mfem_tribol.cpp

```diff
@@ -82,7 +84,19 @@ void setMfemLORFactor( IndexT cs_id, int lor_factor )
   SLIC_ERROR_ROOT_IF( !coupling_scheme->hasMfemData(),
                       "Coupling scheme does not contain MFEM data. "
                       "Create the coupling scheme using registerMfemCouplingScheme() to set the LOR factor." );
+
+  // reset binning proximity based on original LOR factor
+  auto orig_binning_proximity_scale = coupling_scheme->getParameters().binning_proximity_scale;
+  auto orig_lor_factor = coupling_scheme->getMfemMeshData()->GetLORFactor();
```

Comment:
Perhaps I'm not seeing this correctly, but it looks like this is in the routine setMfemLORFactor(), which takes the lor_factor as input. Here, instead, you are getting it from the getter routine. Then, on line 96 you set the lor factor based on the input arg. Can you clarify?